modules/test/playwright/tests/object-web/objectFolder.spec.ts

```diff
@@ -131,4 +131,41 @@ test.describe('Manage object definitions through ViewObjectDefinitions', () => {
 			await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 		}
 	});
+
+	test('can edit object folder label and ERC in the ViewObjectDefinitions page', async ({
+		apiHelpers,
+		modalEditObjectFolderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const objectFolder =
+			await apiHelpers.objectAdmin.postRandomObjectFolder();
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.page
+			.locator('li')
+			.filter({hasText: objectFolder.label['en_US']})
+			.click();
+
+		await viewObjectDefinitionsPage.openObjectFolderActions();
+
+		await viewObjectDefinitionsPage.objectFolderEditLabelAndERCOption.click();
+
+		const editedObjectFolderLabel =
+			'objectFolderLabelEdited' + getRandomInt();
+		const editedObjectFolderERC = 'objectFolderERCEdited' + getRandomInt();
+
+		await modalEditObjectFolderPage.editObjectFolderDetails(
+			editedObjectFolderERC,
+			editedObjectFolderLabel
+		);
+
+		expect(page.getByText(editedObjectFolderLabel).first()).toBeVisible();
+		expect(page.getByText(editedObjectFolderERC)).toBeVisible();
```

Comment:
Hi Dan, I think we can specify this expectation a little by stating where we expect this information. If this information appears in an alert, toast, sidebar or other element, this expectation will return true. But we need to make sure the card header contains the correct information. What do you think about using a function to return the locator value based on the parameter, like:

```
	getObjectFolderCardHeaderERC = (
		objectFolderERC: string,
	) => {
		return this.objectFolderCardHeader
			.getByRole('strong')
			.filter({hasText: objectFolderERC});
	};
```

Reply:
So the expected will be:

```
expect(viewObjectDefinitionsPage
    .getObjectFolderCardHeaderERC(editedObjectFolderERC)
    .getByText(editedObjectFolderERC)
).toBeVisible();
```